spark/v3.4/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestDelete.java

```diff
@@ -335,6 +335,37 @@ public void testDeleteFileThenMetadataDelete() throws Exception {
         sql("SELECT * FROM %s ORDER BY id", selectTarget()));
   }
 
+  @Test
+  public void testDeleteWithPartitionedTable() throws Exception {
```

Comment:
[TestDelete](https://github.com/apache/iceberg/blob/main/spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestDelete.java#L86C7-L86C7) is an abstract test class which extends by both Copy on write and merge on read